README.md

```diff
@@ -49,21 +57,19 @@ Fork local repository from Seagate's CORIO. Clone CORIO repository from your loc
     8. Use following command to install python packages.
             `pip install --ignore-installed -r requirements.txt`
 
-Steps to Configure test scripts for execution in parallel mode.:
-
-    Use following command to execute workloads from different scenarios in parallel.
-
-    python corio.py -ak <access_key> -sk <secret_key> -ep <s3.seagate.com> -ti config/io
 
 **Prerequisite:** 
 
-    A S3 account and access key and secret key should be present to carry out execution. 
-    here test input may be either a single yaml file path i.e. config/s3/s3api/bucket_operations.yaml
-    or
-    A Directory path containing multiple tests data input yaml files those need to be executed in parallel
-    i.e. config/s3/s3api or any other customized directory(i.e. config/s3/io) location may be created containing test data input yaml files and can be passed as argument
+- A s3 account and access key and secret key should be present to carry out execution.
+- Test input may be either a single yaml file path or a directory path containing multiple tests data input yaml files, those need to be executed in parallel.
+  - file path i.e. workload/s3/s3api/bucket_operations.yaml
+  - directory path i.e. workload/s3/s3api or any other customized directory(i.e. workload/s3/io)
+  - refer [yaml structure](docs/YAML_documents/yaml_structure.md) and [sample file](docs/YAML_documents/sample_file.yaml) to create workload.
+
+#### CORIO runner command, help options:
 
-CORIO Runner Help Options:
+- Use following command to execute workloads from different scenarios in parallel.
+  - python corio.py -ak <access_key> -sk <secret_key> -ep <s3.seagate.com> -ti <workload_directory_path>
```

Comment:
```suggestion
    python corio.py -ak <access_key> -sk <secret_key> -ep <s3.seagate.com> -ti <workload_directory_path>
```
Make line 72 as code instead of list item